### PR TITLE
fix to get menu_completed showing up in stathat again

### PR DESCRIPTION
**Pull request by rapala61**
#### What's this PR do?
Fixes stathat logging when sending replies to users.

#### How should this be reviewed?
- 👀 

#### Checklist
- [ ] Tested on staging.
- [ ] Tested locally.

### CODE REVIEW

File: lib/conversation/replies.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ function renderContinueConversationMsg(fn) {
             logger.verbose(`saved user:${args.req.user._id} current_campaign:${args.req.user.current_campaign}`);
             fn(args);
             args.req.user.postDashbotOutgoing(args.replyText);
```
Comment (aaronschachter) [inline lib/conversation/replies.js]: The Dashbot analytics should be tracking the messageTemplate and not the actual rendered message:
screen shot 2017-07-07 at 8 54 04 am

**Comment by aaronschachter** [comment]
If we're sticking with Stathat (which.. is questionable if anyone ever looks at Stathat anymore... cc @jamjensen) we need to manually delete all the Stathat stats we've been logging with message text since that deploy:
https://www.stathat.com/v/stats/EIYS
screen shot 2017-07-07 at 8 57 22 am

**Comment by rapala61** [comment]
@aaronschachter made the update for both loggers.

**Comment by aaronschachter** [comment]
Won't we still see the same behavior for the `renderEndConversationMsg` functions?

**Comment by rapala61** [comment]
Yes we would. Changing those now.

**Comment by rapala61** [comment]
@aaronschachter what about now?

**Comment by jamjensen** [comment]
@aaronschachter Sorry just seeing this. I have not used the stathat reports in sometime. Still interested in seeing the breakdown by message type (ask_quantity, ask_photo, reason_why) so that we're able to look into potential drop-off. If another service does a better job then I'm totally game.

**Pull request merged**